mlflow/utils/autologging_utils/__init__.py

```diff
@@ -327,7 +330,7 @@ class will batch them in order to not increase execution time too much by loggin
 
 
 @contextlib.contextmanager
-def batch_metrics_logger(run_id):
+def batch_metrics_logger(run_id, model_id: Optional[str] = None):
```

Comment:
nit: maybe we can skip type hint here for consistency. Other methods of this class don't use model_id.

Reply:
I think it's fine. It does look ugly but it's better than no type hint.